src/app/core/admin/admin-entity-details/admin-entity-panel-component/admin-entity-panel-component.component.ts

```diff
@@ -1,15 +1,61 @@
-import { Component, Input } from "@angular/core";
+import { Component, Input, OnInit } from "@angular/core";
 import { CommonModule } from "@angular/common";
 import { PanelComponent } from "../../../entity-details/EntityDetailsConfig";
 import { EntityConstructor } from "../../../entity/model/entity";
+import { ColumnConfig } from "app/core/common-components/entity-form/FormConfig";
+import { EntityFieldsMenuComponent } from "app/core/common-components/entity-fields-menu/entity-fields-menu.component";
+import { EntityRegistry } from "app/core/entity/database-entity.decorator";
 
 @Component({
   selector: "app-admin-entity-panel-component",
-  imports: [CommonModule],
+  imports: [CommonModule, EntityFieldsMenuComponent],
   templateUrl: "./admin-entity-panel-component.component.html",
   styleUrl: "./admin-entity-panel-component.component.scss",
 })
-export class AdminEntityPanelComponentComponent {
+export class AdminEntityPanelComponentComponent implements OnInit {
   @Input() config: PanelComponent;
   @Input() entityType: EntityConstructor;
+
+  allFields: ColumnConfig[] = [];
+  // Stores the currently active/selected field IDs to be shown in the panel
```

Comment:
let's generally use jsDoc comments to make these available as tooltips in the IDE, etc.
```suggestion
  /** Stores the currently active/selected field IDs to be shown in the panel */
```